packages/design-system/changelog/unreleased/change-remove-opinionated-components

```diff
@@ -0,0 +1,12 @@
+Change: Remove opinionated components
+
+We've removed the following opinionated components from the ODS and moved them to `web-pkg` instead:
+
+- `OcResource` (now `ResourceListItem`)
```

Comment:
There are three more components about resources:
- `OcResourceLink`
- `OcResourceName`
- `OcResourceSize`

Could you please move and rename those as well?